packages/esm-service-queues-app/src/active-visits/active-visits-row-actions.component.tsx

```diff
@@ -26,7 +26,7 @@ export const ActiveVisitRowActionsCell = ({ queueEntry }: QueueTableCellComponen
 };
 
 export const activeVisitActionsColumn: QueueTableColumn = {
-  headerI18nKey: '',
+  header: translateFrom('@openmrs/esm-service-queues-app', 'actions', 'Actions'),
```

Comment:
You should just be able to use the standard `t` function here, right?  translateFrom would be if the translation is coming from a different esm.

Reply:
I tried that, but unfortunately no. The `t` function is usually obtained from `useTranslation()`, which must be called from a react component. I don't think I'm using the [`translateFrom` function](https://github.com/openmrs/openmrs-esm-core/blob/13067210cba73a461e65345388a887186d9da2e5/packages/framework/esm-translations/src/index.ts#L39) as intended, but I don't see any other function that exposes the `i18n` object. 

An alternative would be to turn `QueueTableColumn` into a function that takes in the `t` function passed in from a react component, but I don't think that's clean either.

@ibacher @denniskigen is there a standard way to define translation strings outside of a react component? When I define them using `translateFrom()`, the strings also don't get picked up by the translation pre-commit git hook.

Reply:
Couldn't you just make the header cells react components, similar to the data cells?

Reply:
Our standard practice with situations like this has been to obtain the `t()` function inside a React component and then pass it in to the place that it's needed. The `t()` function itself isn't React-dependent but the `useTranslation()` hook and the contextual namespace is.

Reply:
Even in a library, we can still require a `TFunction` as a parameter... Especially since the library would have no way of knowing the possible translation strings whatsoever (i.e., I don't even see how we could use the `translateFrom()` function).